dockerfile

```diff
@@ -0,0 +1,15 @@
+#dockerfile
+FROM python:3.9
+
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+
+#directory
+WORKDIR /test
```

Comment:
Suggest to rename working directory to something more generic (might not only use this for testing), maybe `/optimeta`.